src/index.js

```diff
@@ -0,0 +1,327 @@
+    target.classList.add('selected-button');
+    try {
+        resultValue = keyboardElements.find(key => key.keycode === keycode).values[currLang][getCorrespondentCaseKeyboardValue()];
+    }
+    catch (e) {}
+    if (resultValue){
+        insertValueIntoTextArea(resultValue);
+    }
+    else{
+        keyboardElements.find(key => key.keycode === keycode).action(event);
+    }
+}
+
+function getTargetByMouseClick(event){
+    const targetElem = event.target.closest('div');
+    const keycode = Number(targetElem.id.slice(7));
+    return {targetElem, keycode};
+}
+
+function mouseDownClickHandler(event) {
+    const target = getTargetByMouseClick(event);
+    if (target.keycode !== 20) {
+        globalPressActionHandler(event, target.keycode, target.targetElem);
+    }
+    else {
+        capsLockButtonClickAction(target.keycode);
+    }
+}
+
+function mouseUpClickHandler(event) {
+    const target = getTargetByMouseClick(event);
+    if (target.keycode !==20) {
+        doActionOnButton(target.keycode, event);
+        target.targetElem.classList.remove('selected-button');
+    }
+    setFocusToTextArea();
+}
+
+function keyUpHandler(event) {
+    if (event.which !== 20){
+        doActionOnButton(event.which, event);
+        removeSelectedButton(event.which);
+        setFocusToTextArea();
+    }
+}
+
+function doActionOnButton(keycode, event){
+    if (keyboardElements.find(key => key.keycode === keycode).action)
+    {
+        keyboardElements.find(key => key.keycode === keycode).action(event);
+    }
+}
+
+function keyDownHandler(event) {
+    if (event.which !== 20){
+        event.preventDefault();
+        globalPressActionHandler(event, event.which, getKeyboardElementByKeyCode(event.which));
+        setSelectedButton(event);
+    }
+    else {
+        capsLockButtonClickAction(event.which);
+    }
+}
+
+function getKeyboardElementByKeyCode(keycode){
+    return document.getElementById('div_id_' + keycode);
+}
+
+function setSelectedButton(keycode){
+    getKeyboardElementByKeyCode(keycode).classList.add('selected-button');
+}
+
+function removeSelectedButton(keycode) {
+    getKeyboardElementByKeyCode(keycode).classList.remove('selected-button');
+}
+
+function keyBoardLanguageSwitcher(){
+    if (currLang === keyboardLanguages.eng){
+        switchToRuKeyboard();
+    }
+    else {
+        switchToEngKeyboard();
+    }
+}
+
+function switchToEngKeyboard() {
+    currLang = keyboardLanguages.eng;
+    document.querySelectorAll('span.' + keyboardLanguages.eng).forEach(elem => elem.classList.remove('hidden'));
+    document.querySelectorAll("span." + keyboardLanguages.ru).forEach(elem => elem.classList.add('hidden'));
+}
+
+function switchToRuKeyboard() {
+    currLang = keyboardLanguages.ru;
+    document.querySelectorAll("span." + keyboardLanguages.ru).forEach(elem => elem.classList.remove('hidden'));
+    document.querySelectorAll('span.' + keyboardLanguages.eng).forEach(elem => elem.classList.add('hidden'));
```

Comment:
You don't have to create separate function for each language. Just make it single function which takes language as parameter.
You can add 'hidden' class to ALL spans and after that removed it for the accepted language